clusters/minikube/start.sh

```diff
@@ -18,7 +18,7 @@ sudo minikube start --memory=8192 --cpus=4 \
   --vm-driver=${vm_driver} \
   --bootstrapper=kubeadm \
   --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook" \
-  --insecure-registry registry.kube-system.svc.cluster.local
+  --insecure-registry $IMAGE_REPOSITORY_HOST
```

Comment:
You're introducing a new requirement within the FATS API here. I'd urge you to avoid doing so unless absolutely necessary. In that case, it needs to be documented in the readme and applied to every other cluster/registry.

Reply:
I can create a `kube-system` namespace for local docker repository. I just afraid that it would create confusion if someone was trying to debug.